tests/test_redis.py

```diff
@@ -46,16 +49,17 @@ def test_random_port(redisdb_random):
     assert redisdb_random.keys('*') == []
 
 
-redis_proc_old_redis = factories.redis_proc(
-    executable='/usr/local/bin/redis-server',
-    port='?')
-redisdb_old_redis = factories.redisdb('redis_proc_old_redis')
+redis_proc_to_mock = factories.redis_proc(port='?')
 
 
-@pytest.mark.xfail(raises=RedisUnsupported)
-def test_old_redis(redisdb_old_redis):
+def test_old_redis(request):
```

Comment:
Wtf, we can also check ``2.0``, ``1.0``, ``0.1``, ``0.4.1``, etc...

This test is useless in my opinion, we should add information about which version we support to README.

Humans read a documentation, not tests!